app/javascript/dashboard/components/ChatList.vue

```diff
@@ -273,6 +340,8 @@ export default {
 
 <style scoped lang="scss">
 @import '~dashboard/assets/scss/woot';
+@import '~widget/assets/scss/variables.scss';
```

Comment:
This is supposed to be used in widget pack only. Please remove this and use CSS variables from `shared/assets/stylesheets/`

Reply:
https://github.com/chatwoot/chatwoot/pull/3239/commits/84c44831e64e65e2efc240708bb12ceea26e22c2